src/StoreBase.ts

```diff
@@ -71,14 +72,10 @@ export abstract class StoreBase {
 
     // If you trigger a specific set of keys, then it will only trigger that specific set of callbacks (and subscriptions marked
     // as "All" keyed).  If the key is all, it will trigger all callbacks.
-    protected trigger(keyOrKeys?: string|number|(string|number)[]) {
-        let throttleMs: number | undefined;
-        if (this._throttleMs !== undefined) {
-            throttleMs = this._throttleMs;
-        } else {
-            // If the store doens't define any throttling, pick up the default
-            throttleMs = Options.defaultThrottleMs;
-        }
+    protected trigger(keyOrKeys?: KeyOrKeys | undefined) {
```

Comment:
you're changing the signature here
Should remain as 
`protected trigger(keyOrKeys?: KeyOrKeys) {`